lib/src/actions/wait_for_transaction_receipt.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:js_interop';
+
+import 'package:wagmi_flutter_web/src/js/wagmi.js.dart';
+
+/// [Documentation API](https://wagmi.sh/core/api/actions/waitForTransactionReceipt)
+typedef OnReplacedCallback = void Function();
```

Comment:
There are missing [parameters](https://wagmi.sh/core/api/actions/waitForTransactionReceipt#onreplaced).

Maybe we could set that action status to `In progress` in the readme, and finish it in another PR.